website/versioned_docs/version-0.25.0/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -23,7 +23,7 @@ Follow the instructions on this page to:
 import TOCInline from "@theme/TOCInline";
 
 <TOCInline toc={(() => {
-           const trimStart = toc.slice(toc.findIndex((node) => node.id == 'installing-weave-gitops-enterprise')+1);
+           const trimStart = toc.slice(toc.findIndex((node) => node.id == 'instal-weave-gitops-enterprise')+1);
```

Comment:
@yiannistri add one more 'l' to your "install" change

Reply:
Well spotted 👍